client/src/pages/HomePage.tsx

```diff
@@ -1,41 +1,32 @@
 import useAuthContext from '@/hooks/useAuthContext';
 import { useEffect, useState } from 'react';
-import clsx from 'clsx';
 
 import * as littlethingsApi from '@/api-client/littlethingsApi';
-import { options, Post } from '@/api-client/homepageutility';
+import { Post } from '@/api-client/homepageutility';
 import NewLittleThingForm from './NewLittleThingForm';
-import { Button } from '@/components/ui/button';
-import { TrashIcon } from '@radix-ui/react-icons';
+
+import TestComponents from './TestComponents';
 
 const HomePage = () => {
   const [posts, setPosts] = useState<Post[]>([]);
   const { user } = useAuthContext();
-  const [deletable, setDeletable] = useState(false);
   const [create, setCreate] = useState(false);
```

Comment:
i dont think we need state for this and handlers changing this state.
see NewLittleThingsForm.tsx for ref